src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTestBase.java

```diff
@@ -112,10 +112,10 @@ protected Struct createOtherRecord(Schema schema) {
   }
 
   protected void verifySearchResults(Collection<SinkRecord> records, boolean ignoreKey, boolean ignoreSchema) throws IOException {
-    verifySearchResults(records, TOPIC, ignoreKey, ignoreSchema);
+    verifySearchResults(records, TOPIC, null, ignoreKey, ignoreSchema);
   }
 
-  protected void verifySearchResults(Collection<?> records, String index, boolean ignoreKey, boolean ignoreSchema) throws IOException {
+  protected void verifySearchResults(Collection<?> records, String index, String expectedRouting, boolean ignoreKey, boolean ignoreSchema) throws IOException {
```

Comment:
The expected routing is always String ?
I guess we are saying that we do not support int or other types ?

Reply:
No, we can use primitive types and string, this test just use string.